documentation/hiera.md

```diff
@@ -119,6 +117,29 @@ Successful on 2 targets: windows_target, ubuntu_target
 Ran on 2 targets
 ```
 
+### Look up data from plan_hierarchy key
+
+When passed `--plan-hierarchy`, the `lookup` command will look up data from Hiera's
+[plan_hierarchy](#outside-apply-blocks) key. This mimics performing a lookup outside an apply block
+in a Bolt plan.
+
+Because `plan_hierarchy` values are not specific to individual targets this command will perform
+just one lookup and return the bare value. The `--plan-hierarchy` option is cannot be passed in
+addition to `--targets`, `--rerun`, or `--query`.
```

Comment:
```suggestion
Because `plan_hierarchy` values are not specific to individual targets, this command performs
just one lookup and returns the bare value. You can't use the `--plan-hierarchy` option
with the `--targets`, `--rerun`, or `--query` options.
```